src/components/AxisLabel/AxisLabel.jsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { lucidClassNames } from '../../util/style-helpers';
+import { createClass } from '../../util/component-types';
+
+const cx = lucidClassNames.bind('&-AxisLabel');
+
+const {
+	number,
+	string,
+	oneOf,
+	any,
+} = React.PropTypes;
+
+/**
+ * {"categories": ["visualizations", "chart primitives"]}
+ *
+ * Labels for axes.
+ */
+const AxisLabel = createClass({
+	displayName: 'AxisLabel',
+
+	statics: {
+		_lucidIsPrivate: true,
+	},
+
+	propTypes: {
+		/**
+		 * Classes are appended to root element along with existing classes using
+		 * the `classnames` library.
+		 */
+		className: any,
+		/**
+		 * Height of the margin this label should fit into.
+		 */
+		height: number,
+		/**
+		 * Width of the margin this label should fit into.
+		 */
+		width: number,
+		/**
+		 * Zero-based color, defaults to -1 which is black.
+		 */
+		color: number,
```

Comment:
this seems restrictive, should there be a way to pass a literal color both here and to associated charts?